config/monitoring/prometheus/apps/prometheus-configs.yaml

```diff
@@ -314,13 +314,16 @@ data:
       metrics_path: /metrics
       scheme: http
       kubernetes_sd_configs:
-        - role: endpoints
+        - role: pod
           namespaces:
             names:
               - <odh_application_namespace>
+          selectors:
+          - role: pod
+            label: 'app.kubernetes.io/part-of=trainingoperator'
```

Comment:
Are you sure that's the correct label selector?

I would have expected these: https://github.com/opendatahub-io/training-operator/blob/20c6ede9f8c7c629a4ee1d102b9f5975aa3284b6/manifests/rhoai/kustomization.yaml#L32-L33

Reply:
I gathered that label from the training operator pod created using ODH 2.12.1.
By all means it should be `app.kubernetes.io/part-of=training-operator`

We could set the label selector to be `app.kubernetes.io/name=training-operator` instead

Reply:
Yes it seems like `app.kubernetes.io/name=training-operator` would be more specific.